mono/utils/mono-threads-coop.c

```diff
@@ -574,13 +574,13 @@ threads_suspend_policy_getenv (void)
 	return policy;
 }
 
-static MonoThreadsSuspendPolicy threads_suspend_policy = -1;
+static MonoThreadsSuspendPolicy threads_suspend_policy = MONO_THREADS_SUSPEND_UNINITIALIZED;
```

Comment:
Ideally this would not be initialized -- 0 would be correct, but that doesn't work currently for understood reasons.